dists/ultrastardx.appdata.xml

```diff
@@ -3,35 +3,39 @@
   <id>eu.usdx.UltraStarDeluxe</id>
   <metadata_license>CC0-1.0</metadata_license>
   <name>UltraStar Deluxe</name>
-  <summary>Karaoke program that evaluates your performance</summary>
+  <summary>Karaoke for your PC</summary>
   <description>
     <p>UltraStar Deluxe is a free open source karaoke game for your PC. It records the voice of the players and gives points if they hit the notes. Unlike the commercial alternatives, UltraStar Deluxe allows users to add their own songs.</p>
   </description>
   <project_license>GPL-2.0</project_license>
+  <developer id="eu.usdx">
+    <name>UltraStar Karaoke</name>
```

Comment:
Is this really what the AppStream specification wants to have here?

Reply:
I think so? 
https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#developer-name
https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-developer

Reply:
Ok, the second link makes it clear that not only people but also organizations can be named.